dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -287,8 +287,8 @@ def script_level_params(script_level)
 
   test 'should not show concept video for non-legacy script level' do
     non_legacy_script_level = create(:script_level)
-    concept_with_video = Concept.find_by_name('sequence')
-    non_legacy_script_level.level.concepts = [concept_with_video]
+    concept_without_video = Concept.find_by_name('sequence')
+    non_legacy_script_level.level.concepts = [concept_without_video]
```

Comment:
Why the rename here? Seems to me that the concept here _does_ have a video, and what we're testing is that the nature of the level means we ignore that video. If the concept being used here doesn't have a video, then we're not actually testing the thing we're supposed to be testing.

Arguably, we should even have a step in this test asserting that the concept does have a video, in order for the `assert_empty assigns(:level).related_videos` to be meaningful

Reply:
This concept doesn't have a video (the "sequence" concept happens to be the only concept without a video) and it looks like it never had one. I assumed that the variable names were just misleading as a result, after spending a long time trying to track this failure down to find out I had a bug that was return a video for every concept.

[videos.csv](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/videos.csv)

Reply:
hmm, so is this test name misleading then? I interpreted this as "we don't want to show videos for non-legacy script levels, even if those levels are associated with a concept with a video". Is that inaccurate?

Reply:
In that case, I'd recommend leaving this test as-is in _this_ PR, and opening another PR to track either fixing the misleading variable name, the misleading test name, or the test itself if it turns out that's necessary.

Reply:
Reverted. I'll follow up with another PR.